pkgs/racket-doc/scribblings/reference/syntax-util.scrbl

```diff
@@ -141,13 +141,15 @@ the parameter has a non-false value. This is done automatically by
 forms like @racket[with-disappeared-uses].
 }
 
-@defform[(with-disappeared-uses stx-expr)
+@defform[(with-disappeared-uses body-expr ... stx-expr)
```

Comment:
This makes it look like there's an asymmetry between `body-expr`s and `stx-expr`, but the prose seem to consider them indistinguishable. Any reason not to have one name and `...+`? Same with the `#:contracts` below.